.circleci/config.yml

```diff
@@ -54,6 +68,4 @@ workflows:
       - static-code-analysis:
           context:
             - docker-hub-creds-ro
-      - build:
-          context:
```

Comment:
Had to remove this because it shadows the variables added on project level.

Reply:
should we remove it for `static-code-analysis` too?